### DISCUSSION TITLE
What is the correct NPM version to use for @turf/turf?

### DISCUSSION THREAD

**Discussion by aldensc** [Turfjs/turf#2661] [General]
When doing an npm install @turf/turf, I get @turf/turf@7.1.0-alpha.7 installed.  Being an alpha release, I do not know how stable this is.  I think the stable version of turf is listed as 7.0.0.  I know I can install either version, but curious why it installs the alpha version by default and need to know which is the correct version to use (as a user of the library).  Regards - Scott

**Comment by smallsaucepan** [comment]
Hi @aldensc. That shouldn't happen. Just tried it myself and 7.0.0 was installed:

```
$ npm install @turf/turf
$ npm why @turf/turf
@turf/turf@7.0.0
node_modules/@turf/turf
  @turf/turf@"^7.0.0" from the root project
```

Could you please run `npm why @turf/turf` on your computer and share the results?

Is it possible you previously installed the alpha on purpose, and there is still a reference in your package-lock.json file?

**Reply by aldensc** [reply]
Thanks for the reply.  After forcing 7.0.0, uninstalling and installing without a version, it installs 7.0.0.  I'm not sure what I did that caused this.  Maybe like you said, I may have had a reference to it somewhere.  I guess disregard and I'll respond back if it happens again.  Much appreciated!

**Comment by smallsaucepan** [comment]
Glad it worked out!
